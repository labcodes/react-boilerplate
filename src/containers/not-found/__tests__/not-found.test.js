'use strict';

import React from 'react';

// Test helpers
import renderer from 'react-test-renderer';

// Component
import NotFound from 'containers/not-found';

// ====

describe('NotFound [Snapshot]', () => {
    it('Renders correctly', () => {
        const tree = renderer.create(<NotFound />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
