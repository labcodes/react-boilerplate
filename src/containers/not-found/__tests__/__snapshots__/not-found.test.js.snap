// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound [Snapshot] Renders correctly 1`] = `
<h1>
  NotFound
</h1>
`;
